webhook_server/tests/test_runner_handler.py

```
@@ -1889,6 +1909,282 @@ async def run_command_side_effect(command: str, **kwargs: Any) -> tuple[bool, st
+
+    @pytest.fixture
+    def mock_github_webhook(self) -> Mock:
+        """Create a mock GithubWebhook instance."""
+        mock_webhook = Mock()
+        mock_webhook.hook_data = {"action": "opened"}
+        mock_webhook.logger = Mock()
+        mock_webhook.log_prefix = "[TEST]"
+        mock_webhook.repository = Mock()
+        mock_webhook.repository.clone_url = "https://github.com/test/repo.git"
+        mock_webhook.repository.owner.login = "test-owner"
+        mock_webhook.token = "test-token"  # pragma: allowlist secret
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "token"

(S105)